lib/index.js

```diff
@@ -161,10 +161,14 @@ class EvervaultClient {
   async encrypt(data) {
     if (!Datatypes.isDefined(this._derivedAesKey)) {
       if (!Datatypes.isDefined(this._ecdhTeamKey)) {
-        const { ecdhKey } = await this.http.getCageKey();
+        const result = await this.http.getCageKey();
+        const teamKey =
+          this.curve == EvervaultClient.CURVES.PRIME256V1
```

Comment:
Just to be consistent with the rest of the codebase
```suggestion
          this.curve === EvervaultClient.CURVES.PRIME256V1
```